screenpy_selenium/actions/clear.py

```diff
@@ -2,14 +2,16 @@
 
 from __future__ import annotations
 
-from typing import Type, TypeVar
+from typing import TYPE_CHECKING, TypeVar
 
-from screenpy.actor import Actor
 from screenpy.exceptions import DeliveryError
 from screenpy.pacing import beat
 from selenium.common.exceptions import WebDriverException
 
-from ..target import Target
+if TYPE_CHECKING:
+    from screenpy.actor import Actor
+
+    from ..target import Target
 
 SelfClear = TypeVar("SelfClear", bound="Clear")
```

Comment:
You can actually move these `TypeVar` declarations into the `TYPE_CHECKING` block, too, if you want. Then you can also import the `TypeVar` in there.

I don't have strong opinions on it, but i do like that all the type checking stuff is contained within that block that way.

Reply:
I'm about to yank all of those typevars out after the ruff changes.